src/Nethermind/Nethermind.TxPool/TxPoolConfig.cs

```diff
@@ -20,6 +20,7 @@ public class TxPoolConfig : ITxPoolConfig
     {
         public int PeerNotificationThreshold { get; set; } = 5;
         public int Size { get; set; } = 2048;
+        public uint FutureNonceRetention { get; set; } = 16;
```

Comment:
what is Geth's default value?

Reply:
Truth be told I don't see similar functionality in Geth nor OE. We can remove it altogether or keep it.

Reply:
Geth definitely uses it just not sure what number. Martin confirmed that one on a call

Reply:
OE calculates it dynamically to be about 1% per address but not less than 16 txs